spec/test_app_templates/Gemfile.extra

```diff
@@ -1,5 +1,7 @@
 # Use this file to reference specific commits of gems.
 
+gem 'riiif'
```

Comment:
Without this dependency in the application, Rails blows up when attempting to load the above initializer due to a `LoadError` caused by `Riiif::Image`. (Adding `require 'riiif'` to the top of the initializer is not sufficient to work around this error -- the `Riiif::Image` constant remains missing, though curiously enough, `Riiif::Engine` is there. Something something engines loading engines.)